app/views/view_submissions.scala.html

```diff
@@ -37,6 +37,7 @@
   } else {
     @link(
       content = components.elements_in_separate_lines(submission.ucrBlocks.map(_.fullUcr)),
+      textHidden = Some(messages("submissions.hidden.text", submission.ucrBlocks.map(_.fullUcr).mkString)),
```

Comment:
This might not be what you want. Calling `mkString` without any parameters results in glue-ing the elements together. You might rather want to use something like: `.mkString(", ")` to separate the elements with comma.